## Diagnostics

### Abstract

### Importance

### Introduction

Since the emergence of _Severe acute respiratory syndrome-like coronavirus 2_ (SARS-CoV-2) in late 2019, significant international concern has focused on how to manage the spread of the virus.
Identifying individuals who have contracted coronavirus disease 2019 (COVID-19) is crucial to slowing down the global pandemic.
Given the high transmissibility of SARS-CoV-2 and the potential for asymptomatic or presymptomatic individuals to be contagious [@individual-pathogenesis], the development of rapid, reliable, and affordable methods to detect SARS-CoV-2 infection is and was vitally important for understanding and controlling spread.
For instance, test-trace-isolate procedures were an early cornerstone of many nations' efforts to control the outbreak [@doi:10.1503/cmaj.202120; @doi:10/ftzx; @doi:10.1016/j.lanwpc.2020.100044].
<!-- To Do: possibly add text on what goes into developing new diagnostics?-->

The genetic sequence of the virus was first released by Chinese officials on January 10, 2020, and the first test to detect the virus was released about 13 days later [@doi:10.2807/1560-7917.ES.2020.25.3.2000045].
This information is important to the development of diagnostic approaches.
There are two main classes of diagnostic tests: molecular tests, which can diagnose an active infection by identifying the presence of SARS-CoV-2, and serological tests, which can assess whether an individual was infected in the past via the presence or absence of antibodies against SARS-CoV-2.
As the pandemic has evolved throughout 2020 and 2021, a variety of tests have emerged within these two categories.

Molecular tests detect either viral RNA or protein in a patient sample.
Molecular tests are essential to identify infected individuals who may need to seek treatment, quarantine, and whose contacts may need to quarantine.
Tests for viral RNA are done by reverse transcription (RT) of viral RNA to DNA followed by DNA amplification, usually by the polymerase chain reaction (PCR) [@DOI:10.1038/jid.2013.1].
Tests for viral proteins are most often done using detection by an antibody pair in lateral flow tests (LFTs) [@doi:10.1007/s00216-010-3661-4]. 
Molecular tests require the viral genome sequence to develop DNA primers for viral RNA detection, or to express a viral protein for use as an antigen in antibody production.

Serological tests detect the presence of antibodies in blood plasma samples using enzyme-linked immunosorbent assay (ELISA), lateral flow immunoassay (LFIA), and chemiluminescence immunoassay (CLIA) [@DOI:10.3389/fmolb.2021.682405].
Because vaccines are based on the viral spike protein, to distinguish past infection from vaccination, serology tests look for antibodies that bind the nucleocapsid protein of the SARS-CoV-2 virus [@doi:10.1093/infdis/jiaa273]. 
Serological tests are useful for collecting population-level information for epidemiological analysis, as they can be used to estimate the extent of the infection in a given area.
Thus, serological tests may be useful to better understand the percent of cases that manifest as severe versus mild and for guiding public health and economic decisions regarding resource allocation and counter-disease measures.
<!-- To Do: How did we discover the identity of these to start binding them?-->
<!-- To Do: Discuss HCoV broadly and the potential risk of detecting other coronaviruses-->
<!-- To Do: Trade offs in cost vs precision (throughput)-->
<!-- To Do: Discuss viral load vs contagiousness vs symptoms vs antibody development? -->

<!-- To:Write / complete - #### Extant Work

Prior narrative reviews [@doi:10.7326/M20-1301] and systematic reviews [@doi:10.1080/14737159.2020.1816466; @doi:10.1016/j.ajic.2020.07.011] of diagnostic technology for COVID-19 have been perfomed, and these contain detailed review of technologies and specific tests or techniques that exist.
Other reviews provided interim status and reviewed challenges [@doi:10.1016/j.isci.2020.101406], or high level overviews [@doi:10.1126/scitranslmed.abc1931].
Policy discussions have also reviewed both the critical importance of diagnostic in the developing world [@doi:10.1038/d41586-020-01265-0], and how they can best be used [@http://documents.worldbank.org/curated/en/426741613767908998/Five-Ways-that-COVID-19-Diagnostics-Can-Save-Lives-Prioritizing-Uses-of-Tests-to-Maximize-Cost-Effectiveness], and unfortunately, the need for triage when insufficient supplies are available [@doi:10.1016/j.jacc.2020.05.006] -->

### Molecular Tests

Molecular tests are used to identify distinct genomic subsequences of a viral molecule in a sample and thus to diagnose an active viral infection.
An important first step is identifying which biospecimens are likely to contain the virus in infected individuals and then acquiring these samples from the patient(s) to be tested.
Common sampling sources for molecular tests include nasopharyngeal cavity samples, such as throat washes,  throat swabs, and saliva [@doi:10/ggp4qx], and stool samples [@doi:10.1002/jmv.25742].
Once a sample is acquired from a patient, molecular tests detect SARS-CoV-2 based on the presence of either (1) viral nucleic acids or (2) viral protein. 

#### Nucleic Acid Tests
When testing for RNA from viruses like SARS-CoV-2, the first step involves pre-processing in order to create complimentary DNA (cDNA) from the RNA sample using reverse transcription (RT). 
The second step involves the amplification of a region of interest in the cDNA using successive cycles of heating and cooling.
Depending on the application, amplification is achieved using variations of the polymerase chain reaction (PCR).
Nucleic acid tests differ primarily in detection methods, which are detailed below.

##### RT-PCR

<!-- To Do: Check that all of these RT-PCR definitions are consistent, they seem to vary a little across sections... real time or reverse transcriptase?-->
Reverse Transcription Polymerase Chain Reaction (RT-PCR) tests determine whether a target is present by amplifying a region of interest of cDNA [@doi:10.1042/BIO20200034].
The first test developed and validated for the detection of SARS-CoV-2 uses RT-PCR with reverse transcription [@doi:10.2807/1560-7917.ES.2020.25.3.2000045] to detect several regions of the viral genome: the _ORF1b_ of the RNA-dependent RNA polymerase (RdRP), the Envelope protein gene (_E_), and the Nucleocapsid protein gene (_N_).
The publication reporting this text was released on January 23, 2020, less than two weeks after the sequence of the virus was first reported [@doi:10.2807/1560-7917.ES.2020.25.3.2000045].
In collaboration with several other labs in Europe and in China, the researchers confirmed the specificity of this test with respect to other coronaviruses against specimens from 297 patients infected with a broad range of respiratory agents.
Specifically this test utilizes two probes against RdRP, one of which is specific to SARS-CoV-2 [@doi:10.2807/1560-7917.ES.2020.25.3.2000045].
Importantly, this assay was not found to return false positive results.
<!-- To Do: How has this changed in the past year? Are we worried about viral evolution? Mention this approach is specific to SARS-CoV-2 rather than other HCoV-->

##### qPCR

In this manuscript we refer to quantitative real-time PCR as qPCR, following the recommendations of the MIQE guidelines [@doi:10.1373/clinchem.2008.112797]. 
In contrast to RT-PCR, qPCR uses fluorescent dyes that bind to the amplified DNA, thereby allowing a real time assessment of the amplification procedure [@doi:10.1042/BIO20200034].
The time resolution of qPCR is useful because the amount of fluorescence emitted by the sample is proportional to the amount of DNA amplified and it can be indirectly measured using the cycle threshold (C~t~). 
For SARS-CoV-2, studies have typically considered a patient to be infectious if the C~t~ is below 33 [@doi:10.1007/s10096-020-03913-9; @doi:10.1038/s41467-020-20568-4].
Another test was also announced during January 2020.
Chinese researchers developed a reverse transcriptase quantitative real-time PCR (RT-qPCR) test to identify two gene regions of the viral genome, _ORF1b_ and _N_ [@doi:10.1093/clinchem/hvaa029].
<!-- To Do: how does RT-qPCR differ from RT-PCR?-->
This assay was tested on samples from two COVID-19 patients and a panel of positive and negative controls consisting of RNA extracted from several cultured viruses.
The assay uses the _N_ gene to screen patients, while the _ORF1b_ gene region is used to confirm the infection [@doi:10.1093/clinchem/hvaa029].
In this case the test was designed to detect sequences conserved across sarbecoviruses, or viruses within the same subgenus as SARS-CoV-2.
Considering that SARS-CoV-1 and SARS-CoV-2 are the only sarbecoviruses currently known to infect humans, a positive test can be assumed to indicate that the patient is infected with SARS-CoV-2.
However, this test is not able to discriminate the genetics of viruses within the sarbecovirus clade.
<!-- To Do: this means it's probably super robust to any mutations though?-->

##### dPCR

Digital PCR (dPCR) is a new generation of PCR technologies offering an alternative to traditional real-time quantitative PCR.
In dPCR, a sample is partitioned into thousands of compartments, such as nanodroplets (droplet dPCR or ddPCR) or nanowells, and a PCR reaction takes place in each compartment.
This design allows for a digital read-out where each partition is either positive or negative for the nucleic acid sequence being tested for, allowing for much higher throughput.
While dPCR equipment is not yet as common as that for RT-PCR, dPCR for DNA targets generally achieves higher sensitivity than other PCR technologies while maintaining high specificity, though sensitivity is slightly lower for RNA targets [@doi:10.3390/s18041271].
High sensitivity is particularly relevant for SARS-CoV-2 detection, since low viral load in clinical samples can lead to false negatives.
Suo et al. [@doi:10.1080/22221751.2020.1772678] performed a double-blind evaluation of ddPCR for SARS-CoV-2 detection on 57 samples, comprised by 43 samples from suspected positive patients and 14 from supposed convalescents, that had all tested negative for SARS-CoV-2 using RT-PCR.
Despite the initial negative results, 33 out of 35 (94.3%) patients were later clinically confirmed positive.
All of these individuals tested positive using ddPCR.
Additionally, of 14 supposed convalescents who had received two consecutive negative RT-PCR tests, nine (64.2%) tested positive for SARS-CoV-2 using ddPCR.
Two symptomatic patients tested negative with both RT-PCR and ddPCR, but were later clinically diagnosed positive, and 5 of the 14 suspected convalescents tested negative by ddPCR.
While this study did not provide a complete head-to-head comparison to RT-PCR in all aspects, e.g., no samples testing positive using RT-PCR were evaluated by ddPCR, the study shows the potential of dPCR for viral detection even in highly diluted samples.

A second study [@doi:10.1016/j.talanta.2020.121726] confirmed that RT-ddPCR is able to detect SARS-CoV-2 at a lower threshold for viral load relative to RT-PCR.
This study analyzed 196 samples, including 103 samples from suspected patients, 77 from contacts and close contacts, and 16 from suspected convalescents, using both RT-qPCR and RT-ddPCR.
Of the 103 suspected patient samples, RT-qPCR identified 29 as positive, 25 as negative, and 49 as suspected.
Of the RT-qPCR negative or suspected samples, a total of 61 (19 negative and 42 suspected) were confirmed to be positive by RT-ddPCR.
All 103 of the suspected patients were later confirmed to be SARS-CoV-2 positive through a combination of symptom development and RT-qPCR resampling, indicating that RT-ddPCR improved the overall detection rate among these patients from 28.2% to 87.4%.
Of 77 patient samples from contacts and close contacts, 48 tested negative with both methods, and these patients were observed to remain healthy over a period of 14 days.
Within the remaining 29 patient samples, 12 tested positive, 1 negative, and 16 suspected with RT-qPCR.
Fifteen out of 16 suspected results and the negative result were overturned by positive RT-ddPCR results, decreasing the rate of suspected cases from 21% to 1%.
All 16 patients identified as positive by RT-ddPCR were subsequently determined, both clinically and through repeated sampling, to be positive for SARS-CoV-2.
Importantly, all samples that tested positive using RT-qPCR also tested positive using ddPCR.
Among the 16 convalescent patients, RT-qPCR identified 12 as positive, three as suspect, and one as negative, but RT-dPCR identified all 16 as positive.
This evidence further indicates that the lower limit of detection made possible by ddPCR may be useful for identifying when COVID-19 patients are cleared of the virus.
Overall, these studies suggest that ddPCR is a promising tool for overcoming the problem of false-negative SARS-CoV-2 testing.
<!-- To Do: Should we mention the differences in viral load threshhold for these two methods? We could tie this in to how we don't actually know how the viral load relates to infectiousness-->

##### Sequencing
Some tests use the results of the PCR itself to determine whether the pathogen is present, but in other cases, it may be necessary to sequence the amplified DNA.
Sequencing requires an additional sample pre-processing step called library preparation.
Library preparation is the process of preparing the sample for sequencing, typically by fragmenting the sequences and adding adapters [@doi:10.1016/j.biotechadv.2020.107537].
In some cases, library preparation can involve other modifications of the sample, such as adding barcodes to identify a particular sample within the sequence data.
Barcoding can therefore be used to pool samples from multiple sources.
There are different reagents used for library preparation that are specific to identifying one or more target sections with PCR [@doi:10.1021/acsnano.0c02624].
Sequential pattern matching is then used to identify unique subsequences of the virus, and if sufficient subsequences are found, the test is considered positive.
Therefore, tests that utilize sequencing require a number of additional molecular and analytical steps relative to tests that use PCR alone.
Sequencing has been an important strategy for discovery of SARS-CoV-2 variants (for example, see [@DOI:10.1101/2021.01.18.21249786]). 
<!-- To Do: What is the advantage of the sequencing-based tests?-->

##### Pooled and Automated PCR Testing

Due to limited supplies and the need for more tests, several labs have found ways to pool or otherwise strategically design tests to increase throughput.
The first such result came from Yelin et al. [@doi:10.1101/2020.03.26.20039438], who found they could pool up to 32 samples in a single qPCR run.
This was followed by larger-scale pooling with slightly different methods [@doi:10.1101/2020.04.02.022186].
Although these approaches are also PCR based, they allow for more rapid scaling and higher efficiency for testing than the initial PCR-based methods developed.
Technology based on CRISPR (clustered regularly interspaced short palindromic repeats) has also been instrumental in scaling up testing protocols.
<!--To Do: Expand this section to discuss implementation of this strategy using PCR?-->

#### RT-LAMP

RT-PCR remains the gold standard for detection of SARS-CoV-2 RNA from infected patients, but the traditional method requires special equipment and reagents, especially a thermocycler. 
Loop mediated isothermal amplification (LAMP) is an alternative to PCR that does not require a thermocycler [@DOI:10.1093/nar/28.12.e63]. 
LAMP is combined with reverse transcription (RT-LAMP) to enable detection of RNA.
RT-LAMP pilot studies for detection of SARS-CoV-2 were recently reviewed and meta-analyzed [@PMID:34086539].
In the meta analysis of all 2,112 samples, the cumulative sensitivity of RT-LAMP was 95.5% (CI 97.5% = 90.8-97.9%) and the cumulative specificity was 99.5% (CI 97.5% = 97.7-99.9%).
The low cost, excellent sensitivity/specificity, and quick readout of RT-LAMP makes this an attractive alternative to RT-PCR. 
Alternative strategies like RT-LAMP are needed to bring widespread testing away from the lab and into under resourced areas.

One study showed that RT-LAMP is effective for detection of SARS-CoV-2 with excellent specificity and sensitivity, and that this method can be applied to unprocessed saliva samples [@doi:10.1038/s41598-021-95799-6]. 
This test aims to bring the sensitivity of nucleic acid detection to the point of care or home testing setting. 
It could be applied for screening, diagnostics, or as a definitive test for people who are positive based on lateral flow tests.
This method was benchmarked against RT-PCR using 177 human nasopharyngeal RNA samples, of which 126 were COVID positive.
The authors break down the sensitivity of their test according to the cycle threshold (C~t~) value from RT-PCR of the same samples; RT-LAMP performs at 100% sensitivity for samples with a C~t~ from RT-PCR of 32 or less. 
The performance is worse when considering all RT-PCR positive samples (including those with C~t~ values between 32-40).
However, there is some evidence suggesting that samples obtained from individuals that achieve C~t~ values >30 measured using RT-PCR tend to be less infective that those that record a C~t~ value <30 [@doi:10.1016/j.jiph.2021.08.013; @doi:10.1017/ice.2021.132; @doi:10.1093/cid/ciaa619], so RT-LAMP is still a useful diagnostic tool. 
Various combinations of reagents are available, but one example of reagents being used are the WarmStart Colorimetric LAMP 2X Master Mix with a set of six primers developed previously by Zhang et al. [@doi:10.1101/2020.02.26.20028373]. 
To determine assay sensitivity, serial tenfold dilutions of _in vitro_ transcribed N-gene RNA standard (IVT RNA) were tested using quantities from 10^5 copies down to 10 copies. 
The assay readout is the color of the dye changing from pink to yellow due to binding to the DNA product over 30 minutes. 
The RT-LAMP assay was then applied to clinical nasopharyngeal samples.
For viral loads above 100 copies of genomic RNA, the RT-LAMP assay had a sensitivity of 100% and a specificity of 96.1% from purified RNA.
The direct assay of saliva by RT-LAMP had a sensitivity of 85% (CI 70-93%).
Sensitivity and specificity metrics were obtained by comparison with results from RT-PCR.
The estimated cost per test is about 2 euros when RNA extraction is included.
The main strength of this test over RT-PCR is that it can be done isothermally, but the main drawback is that it is about 10-fold less sensitive than RT-PCR.

#### Immunoassays for the Detection of Antigens

Immunoassays can detect molecular indicators of SARS-CoV-2 infection, such as the proteins that act as antigens from the SARS-CoV-2 virus.
<!-- Move this: Since 2020, many immunoassays have been developed for COVID-19 [@url:https://www.centerforhealthsecurity.org/covid-19TestingToolkit/serology/Serology-based-tests-for-COVID-19.html] using several different technologies.-->
They offer the advantage of generally being faster and requiring less specialized equipment than other molecular tests, especially those involving PCR.
As a result, immunoassays hold particular interest for implementation at home and in situations where resources for PCR testing are limited.
The trade-off is that these tests typically have a lower sensitivity, and sometimes a lower selectivity, than other molecular tests.
However, these tests tend to return a positive result five to 12 days after symptom onset, which may therefore correlate more closely with the timeframe during which viral replication occurs [@doi:10.1056/NEJMcp2117115].
Immunoassays for the detection of the SARS-CoV-2 antigen can include LFTs and ELISA, as discussed here, as well as chemiluminescence immunoassays (CLIA) and chromatographic immunoassays [@doi:10.3389/fmed.2021.797109], as described in the serological testing section below.

##### Lateral Flow Tests

Lateral flow tests (LFTs) provide distinct value relative to PCR tests.
They can return results within 30 minutes and can be performed without specialized equipment and at low cost.
They also do not require training to operate, they are cheap to produce, thus they can be distributed widely to affected populations making them an important public health measure to curb pandemic spread. 
LFTs rely on the detection of viral protein with an antibody.
Often this is done with an antibody sandwich format, where one antibody containing a dye binds at one site on the antigen, and an immobilized antibody on the strip binds at another site [@doi:10.1007/s00216-010-3661-4].
This design allows the dye to accumulate to form a characteristic positive test line on the strip [@doi:10.1007/s00216-010-3661-4].
Outside of COVID-19 diagnostics, the applications of LFTs are broad; they are routinely used for home pregnancy tests, disease detection, and even drugs of abuse detection in urine [@DOI:10.1007/s00216-008-2287-2].

A recent review surveyed the performance of LFTs for detection of current SARS-CoV-2 infection [@PMID:34407759].
This review covered 24 studies that included more than 26,000 total LFTs.
They found significant heterogeneity in test sensitivities ranging from 37.7% (95% CI 30.6-45.5) to 99.2% (95% CI 95.5-99.9).
The specificities of these tests were more homogeneous, spanning 92.4% (95% CI 87.5-95.5) to 100.0% (95% CI 99.7-100.0).

Despite having lower sensitivity than PCR tests, LFTs occupy an important niche in the management of SARS-CoV-2.
Current infection detection by LFTs enables the scale and speed of testing that is beneficial to managing viral spread.
LFTs were made available freely to citizens in the United Kingdom at least until December 2021 [@DOI:10.1136/bmj.n1760] and to citizens of the United States in early 2022 [@url:https://www.whitehouse.gov/briefing-room/statements-releases/2022/01/14/fact-sheet-the-biden-administration-to-begin-distributing-at-home-rapid-covid-19-tests-to-americans-for-free].
These tests are particularly useful for ruling out SARS-CoV-2 infection in cases where the likelihood of infection is low (e.g., asymptomatic individuals) and positives (including false positives) can be validated with testing by alternate means [@doi:10/hk34].

##### Enzyme-Linked Immunosorbent Assay

Enzyme-linked immunosorbent assay (ELISA) is a very sensitive immunoassay that can be considered a gold standard for the detection of biological targets, including antibodies and antigenic proteins [@pubmed:32310382].
It can be used to generate either quantitative or qualitative results that can be returned within a few hours [@doi:10.1007/s00253-020-11061-5].
ELISA builds on the idea that antibodies and antigens bind together to form complexes [@pubmed:32310382], utilizing enzymes that react with an antigen as reporter labels [@doi:10.1373/clinchem.2005.051532].
The reagents are immobilized on a plate and anything in the sample that binds is retained [@url:https://www.bosterbio.com/protocol-and-troubleshooting/elisa-principle].
The exact strategy for tagging with a reporter enzyme varies among different types of ELISA [@url:https://www.bosterbio.com/protocol-and-troubleshooting/elisa-principle; @pubmed:32310382].
For COVID-19 diagnostics, ELISAs have been designed to detect all or part of the antigenic Spike protein [@doi:10.12688/wellcomeopenres.15927.1].

One of these assays uses two monoclonal antibodies specific to the nucleocapsid of SARS-CoV-2 to evalaute the relationship between the effect of (estimated) viral load on the ability of the assay to detect the SARS-CoV-2 antigen [@doi;10.1080/14787210.2021.1976144].
This study analyzed naso-oropharyngeal 339 samples that were also analyzed with RT-qPCR, which here was used as a gold standard.
RT-qPCR identified 147 samples as positive and 192 as negative.
They estimated the overall sensitivity and specificity to be 61.9% and 99.0%, respectively.
Sensitivity increased with higher C~t~.
This study also assessed the performance of the ELISA test under different conditions in order to evaluate how robust it would be to the challenges of testing in real-world settings globally.
Higher sensitivity was achieved for samples that were stored under ideal conditions (immediate placement in -80° C).
Therefore, while immediate access to laboratory equipment is an advantage, it is not strictly necessary for ELISA to detect the antigen.

#### CRISPR-based Detection

Technology based on CRISPR (clustered regularly interspaced short palindromic repeats) [@doi:10.1038/s41467-018-04252-2] has also been instrumental in scaling up testing protocols.
Two CRISPR-associated nucleases, Cas12 and Cas13, have been used for nucleic acid detection.
Multiple assays exploiting these nucleases have emerged as potential diagnostic tools for the rapid detection of SARS-CoV-2 genetic material and therefore SARS-CoV-2 infection.
The SHERLOCK method (Specific High-sensitivity Enzymatic Reporter unLOCKing) from Sherlock Biosciences relies on Cas13a to discriminate between inputs that differ by a single nucleotide at very low concentrations [@doi:10.1126/science.aam9321].
The target RNA is amplified by RT-RPA and T7 transcription, and the amplified product activates Cas13a.
The nuclease then cleaves a reporter RNA, which liberates a fluorescent dye from a quencher.
Several groups have used the SHERLOCK method to detect SARS-CoV-2 viral RNA.
An early study reported that the method could detect 7.5 copies of viral RNA in all 10 replicates, 2.5 copies in 6 out of 10, and 1.25 copies in 2 out of 10 runs [@doi:10.1371/journal.ppat.1008705].
It also reported 100% specificity and sensitivity on 114 RNA samples from clinical respiratory samples (61 suspected cases, among which 52 were confirmed and nine were ruled out by metagenomic next-generation sequencing, 17 nCoV-/HCoV+ cases and 36 samples from healthy subjects), and a reaction turnaround time of 40 minutes.
A separate study screened four designs of SHERLOCK and extensively tested the best-performing assay.
They determined the limit of detection to be 10 copies/&mu;l using both fluorescent and lateral flow detection [@doi:10.1101/2020.02.26.967026].
Lateral flow test strips are simple to use and read, but there are limitations in terms of availability and cost per test.
Another group therefore proposed the CREST protocol (Cas13-based, Rugged, Equitable, Scalable Testing), which uses a P51 cardboard fluorescence visualizer, powered by a 9-volt battery, for the detection of Cas13 activity instead of immunochromatography [@doi:10.1128/JCM.02402-20].
CREST can be run, from RNA sample to result, with no need for AC power or a dedicated facility, with minimal handling in approximately 2 hours.
Testing was performed on 14 nasopharyngeal swabs.
CREST picked up the same positives as the CDC-recommended TaqMan assay with the exception of one borderline sample that displayed low-quality RNA.
This approach may therefore represent a rapid, accurate, and affordable procedure for detecting SARS-CoV-2.

The DETECTR method (DNA Endonuclease-Targeted CRISPR Trans Reporter) from Mammoth Biosciences involves purification of RNA extracted from patient specimens, amplification of extracted RNAs by loop-mediated amplification, which is a rapid, isothermal nucleic acid amplification technique, and application of their Cas12-based technology.
In this assay, guide RNAs (gRNAs) were designed to recognize portions of sequences corresponding to the SARS-CoV-2 genome, specifically the N2 and E regions [@doi:10.1038/s41587-020-0513-4].
In the presence of SARS-CoV-2 genetic material, sequence recognition by the gRNAs results in double-stranded DNA cleavage by Cas12, as well as cleavage of a single-stranded DNA molecular beacon.
The cleavage of this molecular beacon acts as a colorimetric reporter that is subsequently read out in a lateral flow assay and indicates the positive presence of SARS-CoV-2 genetic material and therefore SARS-CoV-2 infection.
The 40-minute assay is considered positive if there is detection of both the E and N genes or presumptive positive if there is detection of either of them.
The assay had 95% positive predictive agreement and 100% negative predictive agreement with the US Centers for Disease Control and Prevention SARS-CoV-2 real-time RT–PCR assay.
The estimated limit of detection was 10 copies per &mu;l reaction, versus 1 copy per &mu;l reaction for the CDC assay.
These results have been confirmed by other DETECTR approaches.
Using real-time recombinase polymerase amplification (RT-RPA) for amplification, another group detected 10 copies of synthetic SARS-CoV-2 RNA per &mu;l of input within 60 minutes of RNA sample preparation in a proof-of-principle evaluation [@doi:10.1101/2020.02.29.971127].
Using a similar approach, another group reported detection at 1 copy per &mu;l [@doi:10.1371/journal.pbio.3000978].
The DETECTR protocol was improved by combining RT-RPA and CRISPR-based detection in a one-pot reaction that incubates at a single temperature, and by using dual CRISPR RNAs (which increases sensitivity).
This new assay, known as All-In-One Dual CRISPR-Cas12a (AIOD-CRISPR), detected 4.6 copies of SARS-CoV-2 RNA per &mu;l of input in 40 minutes [@doi:10.1038/s41467-020-18575-6].
Another single-tube, constant-temperature approach using Cas12b instead of Cas12a achieved a detection limit of 5 copies/&mu;l in 40-60 minutes [@doi:10.1038/s41421-020-0174-y].
It was also reported that electric field gradients can be used to control and accelerate CRISPR assays by co-focusing Cas12-gRNA, reporters, and target [@doi:10.1073/pnas.2010254117].
The authors generated an appropriate electric field gradient using a selective ionic focusing technique known as isotachophoresis (ITP) implemented on a microfluidic chip.
They also used ITP for automated purification of target RNA from raw nasopharyngeal swab samples.
Combining this ITP purification with loop-mediated isothermal amplification, their ITP-enhanced assay to achieved detection of SARS-CoV-2 RNA (from raw sample to result) in 30 minutes.

All these methods required upstream nucleic acid amplification prior to CRISPR-based detection. 
They are relying on type V (Cas12-based) and type IV (Cas13-based) CRISPR systems.
In contrast, type III CRISPR systems have the unique property of initiating a signaling cascade, which could boost the sensitivity of direct RNA detection. 
In type III CRISPR systems, guide crRNAs are bound by several Cas proteins [@doi:10.1007/978-1-4939-2687-9_4] and can target both DNA and RNA molecules [@doi:10.1016/j.mib.2017.08.003; @doi:10.1016/j.tim.2016.09.012].
A study tested this hypothesis using the type III-A CRISPR RNA (crRNA)-guided surveillance complex from _Thermus thermophilus_ [@doi:10.1016/j.xcrm.2021.100319]. 
They showed that activation of the Cas10 polymerase generates three products (cyclic nucleotides, protons, pyrophosphates) that can all be used to detect SARS-CoV-2 RNA. 
Detection of viral RNA in patient samples still required an initial nucleic acid amplification step, but improvements may in the future remove that requirement.

This goal of amplification-free detection was later achieved for a Cas13a-based system [@doi:10.1016/j.cell.2020.12.001]. 
This approach combined multiple CRISPR RNAs to increase Cas13a activation, which is detected by a fluorescent reporter. 
Importantly, because the viral RNA is detected directly, the test yields a quantitative measurement rather than a binary result. 
The study also shows that fluorescence can be measured in a custom-made dark box with a mobile phone camera and a low-cost laser illumination and collection optics. 
This makes this approach a truly portable assay for point-of-care diagnostics. 
The authors achieved detection of 100 copies/&mu;l of pre-isolated RNA in 30 minutes, and correctly identified all SARS-CoV-2-positive patient RNA samples tested in 5 minutes (n=20).

There is an increasing body of evidence that CRISPR-based assays offer a practical solution for rapid, low-barrier testing in areas that are at greater risk of infection, such as airports and local community hospitals.
In the largest study to date, DETECTR was compared to qRT-PCR on 378 patient samples [@doi:10.1093/infdis/jiaa641].
The authors reported a 95% reproducibility.
Both techniques were equally sensitive in detecting SARS-CoV-2.
Lateral flow strips showed a 100% correlation to the high-throughput DETECTR assay.
Importantly, DETECTR was 100% specific for SARS-CoV-2 and did not detect other human coronaviruses.
A method based on a Cas9 ortholog from Francisella novicida (FnCas9) achieved 100% sensitivity and 97% specificity in clinical samples, and the diagnostic kit is reported to have completed regulatory validation in India [@doi:10.1016/j.bios.2021.113207].

#### Limitations of Molecular Tests

Tests that identify SARS-CoV-2 using nucleic-acid-based technologies will identify only individuals with current infections and are not appropriate for identifying individuals who have recovered from a previous infection.
Within this category, different types of tests have different limitations.
For example, PCR-based test can be highly sensitive, but in high-throughput settings they can show several problems:
First of all, there is a risk of false-negative responses, which can present a significant problem to large-scale testing.
To reduce occurrence of false negatives, correct execution of the analysis is crucial [@doi:10.1038/d41587-020-00002-2].
Additionally, the emerging nature of the COVID-19 pandemic has introduced some challenges related to Uncertainty surrounding interactions between SARS-CoV-2 and its human hosts.
For example, viral shedding kinetics are still not well understood, but are expected to introduce a significant effect of timing of sample collection on test results [@doi:10.1038/d41587-020-00002-2].
Similarly, the type of specimen could also influence outcomes, as it is not clear which clinical samples are best for detecting the virus [@doi:10.1038/d41587-020-00002-2].
There are also significant practical and logistical concerns.
Much of the technology used for molecular tests is expensive, and while it might be available in major hospitals and/or diagnostic centers, it is often not available to smaller facilities [@doi:10.1126/science.abb8400].
At times during the first year of the pandemic, the availability of supplies for testing, including swabs and testing media, has also been limited [@doi:10.1128/JCM.00512-20].
Similarly, processing times can be long, and tests might take up to 4 days to return results [@doi:10.1126/science.abb8400].
Finally, with CRISPR-based testing strategies, the gRNA can recognize other interspersed sequences on the patient’s genome, false positives and a loss of specificity can occur.
As noted above, false negatives are a significant concern for several reason.
Importantly, clinical reports indicate that it is imperative to exercise caution when interpreting the results of molecular tests for SARS-CoV-2 because negative results do not necessarily mean a patient is virus-free [@doi:10.1128/JCM.00297-20].

### Serological Tests

Although diagnostic tests based on the detection of genetic material can be quite sensitive, they provide information only about active infection, and therefore offer just a snapshot-in-time perspective on the spread of a disease.
Most importantly, they would not work on a patient who has fully recovered from the virus at the time of sample collection.
In this context, serological tests, which use serum to test for the presence of antibodies against SARS-CoV-2, are significantly more informative.
Serological tests can provide insight into population-level dynamics and can also offer a glimpse into the development of antibodies by individual patients during the course of a disease.
Therefore, they can be useful to developing strategies for the management of viral spread.
Furthermore, serological tests hold significant interest because of the possibility that they could provide information relevant to advancing economic recovery and allowing reopenings.
For instance, early in the course of the COVID-19 pandemic, it was hypothesized that people who had developed antibodies might be able to return to work [@doi:10.1101/2020.04.14.20065771], although this strategy would have relied on recovered individuals acquiring long-term immunity.
Some infectious agents can be controlled through "herd immunity", which is when a critical mass within the population acquires immunity through vaccination and/or infection, preventing an infectious agent from spreading widely.
<!--To Do: Because we split out the pathogenesis/transmission section, we need to redefine these concepts-->
A simple SIR model predicts that to achieve the required level of exposure for herd immunity to be effective, at least (1-(1/R~0~)) fraction of the population must be immune or, equivalently, less than (1/R~0~) fraction of the population susceptible [@isbn:9780199209996].
However, for SARS-CoV-2 and COVID-19, the R~0~ and mortality rates that have been observed suggest that relying on herd immunity without some combination of vaccines, proven treatment options, and strong non-pharmaceutical measures of prevention and control would likely result in a significant loss of life.<!--To Do: citation? Scandinavian international comparisons?-->

<~--Include some of this info somewhere?
Understanding the fundamental organization of the human immune response to viral threats is critical to understanding the varied response to SARS-CoV-2.
The human immune system utilizes a variety of innate and adaptive responses to protect against the pathogens it encounters.
The innate immune system consists of barriers, such as the skin, mucous secretions, neutrophils, macrophages, and dendritic cells.
It also includes cell-surface receptors that can recognize the molecular patterns of pathogens.
The adaptive immune system utilizes antigen-specific receptors that are expressed on B and T lymphocytes.
These components of the immune system typically act together; the innate response acts first, and the adaptive response begins to act several days after initial infection following the clonal expansion of T and B cells [@doi:10.1016/j.jaci.2005.09.034].
After a virus enters into a host cell, its antigen is presented by major histocompatibility complex 1 (MHC 1) molecules and is then recognized by cytotoxic T lymphocytes.
-->

#### Sustained Immunity to COVID-19

In the process of mounting a response to a pathogen, the immune system produces antibodies specific to the pathogen.
Understanding the acquisition and retention of antibodies is important both to the diagnosis of prior (inactive) infections and to the development of vaccines.
The two immunoglobulin classes most pertinent to these goals are immunoglobulin M (IgM), which are the first antibodies produced in response to an infection, and immunoglobulin G (IgG), which are the most abundant antibodies found in the blood.
Prior research is available about the development of antibodies to _Severe acute respiratory syndrome-related coronavirus 1_ (SARS-CoV-1) during the course of the associated disease, severe acute respiratory syndrome (SARS).
Following SARS infection, IgM and IgG antibodies were detected in the second week post-infection.
IgM titers peaked by the first month post-infection, and then declined to undetectable levels after day 180.
IgG titers peaked by day 60, and persisted in all donors through the two-year duration of study [@doi:10.1111/j.1440-1843.2006.00783.x].
A two-year longitudinal study following convalesced SARS patients with a mean age of 29 found that IgG antibodies were detectable in all 56 patients surveyed for at least 16 months, and remained detectable in all but 4 (11.8%) of patients through the full two-year study period [@doi:10.1086/500469].
These results suggest that immunity to SARS-CoV-1 is sustained for at least a year.

The persistence of antibodies to SARS-CoV-2 remains under investigation.
Circulating antibody titers to other coronaviruses have been reported to decline significantly after 1 year [@doi:10.1017/s0950268800048019].
Autopsies of lymph nodes and spleens from severe acute COVID-19 patients showed a loss of T follicular helper cells and germinal centers that may explain some of the impaired development of antibody responses [@doi:10.1016/j.cell.2020.08.025].
An early study (initially released on _medRxiv_ on February 25, 2020) presented a chemiluminescence immunoassay to a synthetic peptide derived from the amino acid sequence of the SARS-CoV-2 _S_ protein [@doi:10.1093/infdis/jiaa243].
This method was highly specific to SARS-CoV-2 and detected IgM in 57.2% and IgG in 71.4% and 57.2% of sera samples from 276 confirmed COVID-19 patients.
It reported that IgG could be detected within two days of the onset of fever but that IgM could not be detected any earlier, a pattern they compared to findings in another disease caused by a HCoV, Middle East respiratory syndrome (MERS).
Since then, several trials have reported the potential protective effect of antibodies in convalescent plasma obtained from recovered COVID-19 patients to treat critically ill COVID-19 patients [@doi:10.1172/jci138745; @doi:10.1001/jama.2020.4783; @doi:10.1056/NEJMoa2031893].

Evidence to date suggests that sustained immunity to the SARS-CoV-2 virus remains for a period of at least 6 to 8 months [@doi:10.3201/eid2703.204543; @doi:10.1126/science.abf4063; @pmid:33443036; @doi:10.1016/j.medj.2021.02.001].
Dan et al. assessed sustained immunity using 254 blood samples from 188 COVID-19 positive patients [@doi:10.1126/science.abf4063]. 
The samples were collected at various time points between 6 and 240 days post-symptom onset, meaning some patients were assessed longitudinally. 
Of the samples, 43 were collected at least 6 months after symptom onset.
After 1 month, 98% of patients were seropositive for IgG to the spike protein, S. 
Moreover, S IgG titers were stable and heterogeneous among patients over a period of 6 to 8 months post-symptom onset, with 90% of subjects seropositive at 6 months.
Similarly, at 6 to 8 months 88% of patients were seropositive for receptor binding domain (RBD) IgG and 90% were seropositive for SARS-CoV-2 neutralizing antibodies.

The findings of Dan et al. are in accordance with a study by Sherina et al. that examined 119 samples from 88 donors who had recovered from mild to severe cases of COVID-19 [@doi:10.1016/j.medj.2021.02.001]. 
They observed a relatively stable level of IgG and plasma neutralizing antibodies up to 6 months post diagnosis.
Significantly lower but considerable levels of anti-SARS-CoV-2 IgG antibodies were still present in 80% of samples obtained 6-8 months post symptom-onset.
A study by Gaebler et al. also found that titers of IgM and IgG antibodies against the RBD decreased from 1.3 to 6.2 months post infection in a study of 87 individuals [@doi:10.1038/s41586-021-03207-w]. 
However, the decline of IgA (15%) activity was less pronounced than that of IgM (53%) or IgG (32%).
It was noted that higher levels of anti-RBD IgG and anti-N total antibodies were detected in individuals that reported persistent post-acute symptoms at both study visits.<!--To Do: confirm we define N in this paper-->
Moreover, plasma neutralizing activity decreased five-fold between 1.3 and 6.2 months in an assay of HIV-1 virus pseudotyped with SARS-CoV-2 S protein, and this neutralizing activity was directly correlated with IgG anti-RBD titers [@doi:10.1038/s41586-021-03207-w].
These findings are in accordance with other studies that show that the majority of seroconverters have detectable, albeit decreasing, levels of neutralizing antibodies at least 3-6 months post infection [@doi:10.1126/science.abd7728; @doi:10.1038/s41564-020-00813-8; @doi:10.1038/s41467-021-21444-5].
Determining the potency of anti-RBD antibodies early in the course of an infection may be important moving forward, as their neutralizing potency may be prognostic for disease severity and survival [@doi:10.1016/j.cell.2020.12.015].

The development of memory B cells and memory T cells has also been assessed in several studies.
Dan et al. showed that SARS-CoV-2 S-specific memory B cell levels rose steadily over the first 120 days following symptom onset [@doi:10.1126/science.abf4063].
RBD-specific memory B cells had been detected in COVID-19 patients 90 days post-symptom onset in previous studies [@doi:10.1016/j.cell.2020.11.029; @pmid:33443036], and this study confirms these finding and shows that levels of these cells increased over the 4-5 months post-symptom onset [@doi:10.1126/science.abf4063].
Gaebler et al. have shown that memory B cells specific to the RBD remain unaltered and exhibit clonal turnover and antibody sequence evolution 6 months post infection, indicative of prolonged germinal cell reactions [@doi:10.1038/s41586-021-03207-w]. 
The same study showed that antibodies expressed by these memory B cells have resistance to RBD mutations, greater somatic hypermutations, and increased potency, which the authors suggest might be evidence of continued evolution of humoral immunity [@doi:10.1038/s41586-021-03207-w].
Indeed, Wheatley et al. showed that S-specific IgG^+^ memory B cells consistently increase over time and by 4 months comprise approximately 0.8% of all IgG^+^ memory B cells, which may indicate cellular immune memory to even mild-to-moderate COVID-19 infection [@doi:10.1038/s41467-021-21444-5]. 
Dan et al. showed that N-specific memory B cells steadily increased up to 4-5 months post-symptom onset [@doi:10.1126/science.abf4063].
SARS-CoV-2 memory CD8^+^ T cells were also detected in 70% of 169 COVID-19 patients after 1 month [@doi:10.1126/science.abf4063], which is consistent with previous research [@doi:10.1016/j.cell.2020.05.015].
However, SARS-CoV-2 memory CD8^+^ T cells were slightly decreased (50%) 6 months post-symptom onset. 
In this same subset of COVID-19 patients, 93% of subjects had detectable levels of SARS-CoV-2 memory CD4^+^ T cells, of which 42% had more than 1% SARS-CoV-2-specific CD4^+^ T cells.
At 6 months, 92% of patients were positive for SARS-CoV-2 memory CD4^+^ T cells.
Indeed, the abundance of S-specific memory CD4^+^ T cells over time was similar to that of SARS-CoV-2-specific CD4^+^ T cells overall [@doi:10.1126/science.abf4063].
T cell immunity to SARS-CoV-2 at 6 to 8 months following symptom onset has also been confirmed by other studies [@doi:10.1101/2020.11.01.362319; @doi:10.1101/2020.12.08.416636; @doi:10.1016/j.medj.2021.02.001].
In another study, T cell reactivity to SARS-CoV-2 epitopes was also detected in some individuals never been exposed to SARS-CoV-2. 
This finding suggests the potential for cross-reactive T cell recognition between SARS-CoV-2 and pre-existing circulating HCoV that are responsible for the “common cold” [@doi:10.1016/j.cell.2020.05.015], but further research is required.

Concerns have also been raised that immunity may wane over time.
Several reported cases of reinfection have been confirmed via genomic analysis that revealed distinct variants of SARS-CoV-2 within a single patient [@doi:10/ghfgkt; @doi:10.1093/cid/ciaa1451; @doi:10.1093/cid/ciaa1275].
Further research is required to determine the full extent to which sustained immunity can be and is typically achieved following SARS-CoV-2 infection.
Furthermore, it is unclear whether previous infection may carry repercussions in terms of disease severity for patients [@doi:10/fscx] and what implications the possibility of reinfection holds for vaccine development and the long-term efficacy of vaccines [@doi:10.1038/s41591-020-1121-z; @doi:10.3389/fimmu.2020.571481].
<!--To Do: add https://www.nature.com/articles/s41591-020-0965-6#Sec9 and evalute reports of recurrence.-->

#### ELISA

The application of ELISA to serological testing is complementary to its use in molecular diagnostics (above).
Instead of using an enzyme-labeled antibody as a probe that binds to the target antigen, instead the probe is an antigen and the target is an antibody.
On March 18, 2020, the Krammer lab of the Icahn School of Medicine at Mount Sinai proposed an ELISA test that detects IgG and IgM that react against the RBD of the spike proteins (S) of the virus [@doi:10.1101/2020.03.17.20037713; @doi:10.1038/s41591-020-0913-5].
A subsequent ELISA test developed to detect SARS-CoV-2 IgG based on the RBD have reported a selectivity of over 99% and a sensitivity of up to 88.24%, which was observed in samples collected 21 to 27 days after the onset of infection (approximated with symptom onset or positive PCR test) [@doi:10.3389/fmicb.2020.618097].
Earlier in the disease course, sensitivity was lower: 53.33% between days 0 and 13 and 80.47% between days 14 and 20.
This study reported that their laboratory ELISA outperformed two commercial kits that also used an ELISA design [@doi:10.3389/fmicb.2020.618097].
Therefore, while analysis with ELISA requires laboratory support and equipment, these results do suggest that ELISA achieves relatively high sensitivity, especially in the weeks following infection.
Efforts have been made to develop low-cost strategies for conducting these tests that will make them more accessible world wide [@doi:10.1016/j.jim.2021.113182].

#### Chemiluminescence Immunoassay

Another early approach investigated for detection of antibodies against SARS-CoV-2 was chemiluminescence immunoassay (CLIA).
Like ELISA, CLIA is a type of enzyme immunoassay (EIA) [@doi:10.2147/IJN.S200556].
While the technique varies somewhat, in one approach, a bead is coated with the antigen and then washed with the sample [@doi:10.1007/s13317-017-0097-2].
If the antibody is present in the sample, it will bind to the bead.
Then the bead is exposed to a label, a luminescent molecule that will bind to the antigen/antibody complex and can therefore be used as an indicator [@doi:10.1007/s13317-017-0097-2].
A preprint describing a CLIA approach to identify COVID-19 cases was released on _medRxiv_ on February 25, 2020 [@doi:10.1093/infdis/jiaa243; @doi:10.1101/2020.02.22.20026617].
This test used a synthetic peptide derived from the amino acid sequence of the SARS-CoV-2 _S_ protein [@doi:10.1093/infdis/jiaa243].
It was highly specific to SARS-CoV-2 and detected IgM in 57.2% and IgG in 71.4% of sera samples from 276 COVID-19 cases confirmed with RT-qPCR.
IgG could be detected within two days of the onset of fever, but IgM could not be detected any earlier [@doi:10.1093/infdis/jiaa243], which has been supported by other analyses as well [@doi:10.1016/j.jcv.2020.104690]; this pattern was consistent with observations in Middle East respiratory syndrome (MERS), which is also caused by an HCoV.
In comparisons of different commercial immunoassays, accuracy of CLIA tests were often roughly comparable to other EIAs [@doi:10.1016/j.jcv.2020.104511], although one CLIA did not perform as well as several other EIAs [@doi:10.1016/j.jcv.2020.104690; @doi:10.1128/JCM.01361-20].
The sensitivity and selectivities reported vary among CLIA tests and for the detection of IgM versus IgG, but sensitivities and selectivities as high as 100% have been reported among various high-throughput tests [@doi:10.1002/jmv.25932; @doi:10.1128/JCM.01361-20; @doi:10.1016/j.jcv.2021.104914].
CLIA has previously been used to develop tests that can be used at point of care (e.g., [@doi:10.2147/IJN.S200556]) which may allow for this technique to become more widely accessible in the future.

#### Lateral Flow Immunoassay

The first serological test approved for emergency use in the United States was developed by Cellex [@url:https://www.fda.gov/media/136625/download].
The Cellex qSARS-CoV-2 IgG/IgM Rapid Test is a chromatographic immunoassay, also known as a lateral flow immunoassay, designed to qualitatively detect IgM and IgG antibodies against SARS-CoV-2 in the plasma (from a blood sample) of patients suspected to have developed a SARS-CoV-2 infection [@url:https://www.fda.gov/media/136625/download].
The Cellex test cassette contains a pad of SARS-CoV-2 antigens and a nitrocellulose strip with lines for each of IgG and IgM, as well as a control (goat IgG) [@url:https://www.fda.gov/media/136625/download].
In a specimen that contains antibodies against the SARS-CoV-2 antigen, the antibodies will bind to the strip and be captured by the IgM and/or IgG line(s), resulting in a change of color [@url:https://www.fda.gov/media/136625/download].
With this particular assay, results can be read within 15 to 20 minutes [@url:https://www.fda.gov/media/136625/download].
LFIs are often available at point of care but can have very low sensitivity [@doi:10.1128/JCM.01361-20].

#### Neutralizing Antibody Assays

Neutralizing antibody assays play a functional role in understanding immunity that distinguishes them from other serological tests.
The tests described above are all binding antibody tests.
On the other hand, rather than simply binding an antibody to facilitate detection, neutralizing antibody determines whether an antibody response is present that would prevent infection [@doi:10.1128/Spectrum.01202-21; @doi:10.1038/s41467-020-17892-0].
Therefore, these tests serve the purpose of evaluating the extent to which a sample donor has acquired immunity that will reduce susceptibility to SARS-CoV-2.
As a result, neutralizing antibody assays have been used widely to characterize the duration of immunity following infection, to assess vaccine candidates, and to establish correlates of protection against infection and disease [@doi:10.1038/s41591-021-01377-8; @doi:10.1128/JCM.02107-20; @doi:10/gnx76z].
These tests are typically performed in a laboratory [@doi:10.1128/Spectrum.01202-21], and in SARS-CoV-2, the results of neutralizing antibody assays are often correlated with the results of binding antibody tests [@doi:10.1128/Spectrum.01202-21].

The gold standard for assessing the presence of neutralizing antibodies is the plaque reduction neutralization test (PRNT), but this approach does not scale well [@doi:10.1038/s41467-020-17892-0].
An early high-throughput neutralizing antibody assay designed against SARS-CoV-2 used a fluorescently labeled reporter virus that was incubated with different dilutions of patient serum [@doi:10.1038/s41467-020-17892-0].
The cells used for incubation would turn green if antibodies were not present.
Essentially, this assay evaluates whether the virus is able to infect the cell in the presence of the serum.
The specificity of this assay was 100%, and the correlation between the results of this assay and of PRNT was 0.85 with the results suggesting that the sensitivity of the high-throughput approach was higher than that of PRNT [@doi:10.1038/s41467-020-17892-0].
While this approach was performed on a plate and using cells, other methods have been developed using methods such as bead arrays [@doi:10.1016/j.nbt.2021.10.002].

#### Limitations of Serological Tests

Like molecular tests, serological tests carry a number of limitations that influence their utility in different situations.
Importantly, false positives can occur due to cross-reactivity with other antibodies according to the clinical condition of the patient [@url:https://www.fda.gov/media/136625/download].
Therefore, such tests must be used in combination with RNA detection tests if intended for diagnostic purposes, and while serological tests may be of interest to individuals who wish to confirm they were infected with SARS-CoV-2 in the past, their potential for false positives means that they are not currently recommended for this use.
<!-- To Do: These are also single-throughput (can't be multiplexed), or can they also be high-throughput??-->
Due to the long incubation times and delayed immune responses of infected patients, serological tests are insufficiently sensitive for a diagnosis in the early stages of an infection.
The limitations due to timing make serological tests far less useful for enabling test-and-trace strategies.
<!--To Do: Add a sentence explaining the utility of serosurveys?-->

### Possible Alternatives to Current Practices for Identifying Active Cases

COVID-19 can present with symptoms similar to other types of pneumonia, and symptoms can vary widely among COVID-19 patients; therefore, clinical presentation is often insufficient as a sole diagnostic criterion.
In addition, identifying and isolating mild or asymptomatic cases is critical to efforts to manage outbreaks.
Even among mildly symptomatic patients, a predictive model based on clinical symptoms had a sensitivity of only 56% and a specificity of 91% [@doi:10.2807/1560-7917.ES.2020.25.16.2000508].
More problematic is that clinical symptom-based tests are only able to identify already symptomatic cases, not presymptomatic or asymptomatic cases.
They may still be important for clinical practice, and for reducing tests needed for patients deemed unlikely to have COVID-19.

Similarly, X-ray diagnostics have been reported to have high sensitivity but low specificity in some studies [@doi:10.1148/radiol.2020200642].
Other studies have shown that specificity varies between radiologists [@doi:10.1148/radiol.2020200823], though the sensitivity reported here was lower than that published in the previous paper.
However, preliminary machine-learning results have shown far higher sensitivity and specificity from analyzing chest X-rays than was possible with clinical examination [@doi:10.1007/s13246-020-00865-4].
X-ray tests with machine learning can potentially detect asymptomatic or presymptomatic infections that show lung manifestations.
This approach would still not recognize entirely asymptomatic cases.
Given the above, the widespread use of X-ray tests on otherwise healthy adults is likely inadvisable.

### Strategies and Considerations for Determining Whom to Test

Early in the COVID-19 pandemic, testing was typically limited to individuals considered high risk for developing serious illness [@url:https://www.cdc.gov/coronavirus/2019-nCoV/hcp/clinical-criteria.html].
This approach often involved limiting testing to people with severe symptoms and people showing mild symptoms that had been in contact with a person who had tested positive.
Individuals who were asymptomatic (i.e., potential spreaders) and individuals who were able to recover at home were therefore often unaware of their status.
However, this method of testing administration misses a high proportion of infections and does not allow for test-and-trace methods to be used.
For instance, a recent study from Imperial College estimates that in Italy, the true number of infections was around 5.9 million in a total population ~60 million, compared to the 70,000 detected as of March 28th [@doi:10.25561/77731].
Another analysis, which examined New York state, indicated that as of May 2020, approximately 300,000 cases had been reported in a total population of approximately 20 million [@url:https://www.governor.ny.gov/sites/governor.ny.gov/files/atoms/files/NYForwardReopeningGuide.pdf].
This corresponded to ~1.5% of the population, but ~12% of individuals sampled statewide were estimated as positive through antibody tests (along with indications of spatial heterogeneity at higher resolution) [@url:https://www.governor.ny.gov/sites/governor.ny.gov/files/atoms/files/NYForwardReopeningGuide.pdf].
Technological advancements that facilitate widespread, rapid testing will therefore improve the potential to accurately assess the rate of infection and aid in controlling the virus' spread.
<!-- To Do: update this section because there have plenty of efforts to do this in the past 12 months, e.g., countries, universities, etc.-->

### What lies ahead
Diagnostic tools have played an important role during the COVID-19 pandemic. 
Specifically, the results of SARS-CoV-2 diagnostic tests (typically qPCR or LFT-based tests) have been used to estimate the number of infections in the general population, thus informing public health strategies around the globe [@doi:10.1038/s41576-021-00360-w]. 
During the surges caused by the different SARS-CoV-2 variants between 2020 and 2021, government-sponsored efforts to conduct mass testing and to provide free diagnostic tests to the population were a common occurrence in many parts of the world [@doi:10.1136/bmj.n902; @doi:10.1177/0020852321997552; @doi:10.1016/j.worlddev.2020.105208]. 
However, recent reports indicate that such public health policies are starting to change during 2022. 
For example, it is known that the UK plans to dismantle its COVID-19 testing program and scale back its daily reporting requirements [@url:https://www.independent.co.uk/news/health/who-covid-testing-anil-soni-b2032884.html; @doi:10.1136/bmj.o562]. 
A similar approach can be seen in the US as well, where multiple state-run testing facilities are closing, despite some groups advocating to keep them open [@url:https://www.9news.com/article/news/health/coronavirus/local-groups-continue-push-for-covid-testing-and-vaccinations/73-bbcd8384-d96a-425e-aaeb-16ac9f36e581; @url:https://www.sltrib.com/news/2022/02/18/utah-will-stop-daily].
These ongoing changes in testing policy are likely to have a direct effect on how the pandemic is managed moving forward, because SARS-CoV-2 diagnostic tests can be used effectively to slow the spread of the disease only when 1) they are used to share testing results in a timely manner so that they can reasonably be used to approximate the number of infections in the population, and 2) those tests are easily accessible by the general public. 
In this regard, it is important to mention that the multiple SARS-CoV-2 diagnostic tools we have presented through this paper lack any real benefit if they are difficult to obtain, or if their use on a limited basis results in biased data that would lead to ill-informed public health strategies. 
Because the pandemic is still ongoing, the fact that active testing and tracing efforts seem not to be a priority anymore for the public health apparatus in different countries is a worrying trend that if continued, could result in increased morbidity and mortality and an overall failure to end the pandemic. 

### Conclusions

<!-- To Do: This section has less discussion than other sections, not sure if we want to try to expand it?
Potentially mention that there are longitudinal studies on how long sustained immunity lasts, but time makes it easier to get an idea of the distribution without much time elapsing-->
Major advancements have been made in identifying diagnostic approaches.
The development of diagnostic technologies have been rapid, beginning with the release of the SARS-CoV-2 viral genome sequence in January.
As of October 2020, a range of diagnostic tests have become available.
One class of tests uses PCR (RT-PCR or RT-qPCR) to assess the presence of SARS-CoV-2 RNA, while another typically uses ELISA to test for the presence of antibodies to SARS-CoV-2.
The former approach is useful for identifying active infections, while the latter measures hallmarks of the immune response and therefore can detect either active infections or immunity gained from prior infection.
Combining these tests leads to extremely accurate detection of SARS-CoV-2 infection (98.6%), but when used alone, PCR-based tests are recommended before 5.5 days after the onset of the illness and antibody tests after 5.5 days [@doi:10.1001/jama.2020.8259].
Other strategies for testing can also influence the tests' accuracy, such as the use of nasopharyngeal swabs versus bronchoalveolar lavage fluid [@doi:10.1001/jama.2020.8259], which allow for trade-offs between patient's comfort and test sensitivity.
Additionally, technologies such as digital PCR may allow for scale-up in the throughput of diagnostic testing, facilitating widespread testing.
One major question that remains is whether people who recover from SARS-CoV-2 develop sustained immunity, and over what period this immunity is expected to last.
Some reports have suggested that some patients may develop COVID-19 reinfections (e.g., [@doi:10/ghfgkt]), but the rates of reinfection are currently unknown.
Serologic testing combined with PCR testing will be critical to confirming purported cases of reinfection and to identifying the duration over which immunity is retained and to understanding reinfection risks.
